### DISCUSSION TITLE
Group collections

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [directus/directus#9154] [Feature Requests]
It would be nice to be able to group collections and making "subcollection"
For example something like this screen:
<img width="177" alt="Schermata 2021-10-26 alle 14 39 15" src="https://user-images.githubusercontent.com/63153191/138880452-ea180aff-b0ca-43a9-be64-770d8e1b2fed.png">
In this case categories are used only by articoli (posts), and with this view i think it would be more clear for the user

**Comment by joselcvarela** [comment]
Hello @federicocappellotto97 
This is already possible
![image](https://user-images.githubusercontent.com/14039341/138886447-a98656de-ea82-4479-8f4f-bc4f3aa28cb2.png)

For that you will need the Directus rc99.
Then on your "Data Model" menu you can drag n' drop your models, including nested models.

**Reply by federicocappellotto97** [reply]
I didn't know!!! Thank you so much man!
